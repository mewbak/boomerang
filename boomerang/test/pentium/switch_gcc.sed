s/134517890/"Other!\\n"/;
s/134517848/"Two!\\n"/;
s/134517854/"Three!\\n"/;
s/134517862/"Four!\\n"/;
s/134517869/"Five!\\n"/;
s/134517876/"Six!\\n"/;
s/134517882/"Seven!\\n"/;
