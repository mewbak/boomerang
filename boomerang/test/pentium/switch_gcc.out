
int main(int argc, char** argv, int param4)
{
int local2;
int local5;
    if (argc - 2 > 5) {
        local2 = 134517890;
    } else {
        switch(local0) {
        case 2:
            local2 = 134517848;
        break;
        case 3:
            local2 = 134517854;
        break;
        case 4:
            local2 = 134517862;
        break;
        case 5:
            local2 = 134517869;
        break;
        case 6:
            local2 = 134517876;
        break;
        case 7:
            local2 = 134517882;
        break;
        }
    }
    printf(local2, local5, param4, argc, argv);
    return 0;
}

